src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -52,9 +52,14 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
      */
     public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
+        setTagManager(persons);
     }
 
-    public void setTagManager(List<Person> persons) {
+    /**
+     * Resets the TagManager to contain only the new tagging in the given {@code persons} list.
```

Comment:
new tagging sounds a bit weird, but this could be me. could you elaborate on what you mean?

Reply:
This was supposed to be 'contains only tag-person mappings from the `persons` list.' The original message was not written very well. Thanks!